lib/matplotlib/pylab.py

```diff
@@ -231,15 +229,15 @@
 ## We are still importing too many things from mlab; more cleanup is needed.
 
 from matplotlib.mlab import (
-    amap, base_repr, binary_repr, bivariate_normal, center_matrix, csv2rec,
```

Comment:
Everything removed from here should also go in the deprecation list. I suspect that the chance that someone may still be using these ones is higher than the `mlab` ones, even.